<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionneursClass</class>
 <widget class="QMainWindow" name="ActionneursClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Actionneurs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="nlblConnect">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConnect">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnecte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDConnect">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay1ON">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay1OFF">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay2OFF">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay2ON">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay3OFF">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay3ON">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay4OFF">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRelay4ON">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="relay4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Relay 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status : OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status : OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status : OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status : OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblError">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Actionneurs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDConnect</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>onDisconnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay1ON</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay1ON()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay1OFF</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay1OFF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay2ON</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay2ON()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay2OFF</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay2OFF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay3ON</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay3ON()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay3OFF</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay3OFF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay4ON</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay4ON()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelay4OFF</sender>
   <signal>clicked()</signal>
   <receiver>ActionneursClass</receiver>
   <slot>Relay4OFF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onDisconnectButtonClicked()</slot>
  <slot>Relay1ON()</slot>
  <slot>Relay2ON()</slot>
  <slot>Relay3ON()</slot>
  <slot>Relay4ON()</slot>
  <slot>Relay1OFF()</slot>
  <slot>Relay2OFF()</slot>
  <slot>Relay3OFF()</slot>
  <slot>Relay4OFF()</slot>
 </slots>
</ui>
